packages/@react-spectrum/s2/stories/ComboBox.stories.tsx

```diff
@@ -210,3 +211,11 @@ ContextualHelpExample.parameters = {
     }
   }
 };
+
+export const CustomWidth = (args: any) => <Example {...args} styles={style({width: 384})} />;
```

Comment:
just a heads up for the future, I'm pretty sure the way Storybook wants you to extend these is to instead do something like 
```
export const CustomWidth = {
  ...Example
  args: {
    ...Example.args,
    styles: style({width: 384})
  }
}
```

Reply:
jsx is so much more readable though. what's the advantage of this?